documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1900,80 +2072,90 @@ database.history.consumer.ssl.key.password=test1234
 
 Be sure to consult the {link-kafka-docs}.html[Kafka documentation] for all of the configuration properties for Kafka producers and consumers. (The SQL Server connector does use the {link-kafka-docs}.html#newconsumerconfigs[new consumer].)
 
+
 // Type: assembly
-// ModuleID: updates-to-refresh-capture-tables-after-a-schema-change
-// Title: Updates required to refresh capture tables after a schema change 
+// ModuleID: refreshing-capture-tables-after-a-schema-change
+// Title: URefreshing capture tables after a schema change 
 [[sqlserver-schema-evolution]]
 === Database schema evolution
 
-{prodname} is able to capture schema changes over time.
-Due to the way CDC is implemented in SQL Server, it is necessary to work in co-operation with a database operator in order to ensure the connector continues to produce data change events when the schema is updated.
+When change data capture is enabled for a SQL Server table, as changes occur in the table, event records are persisted to a capture table on the server.
+If you introduce a change in the structure of the source table change, for example, by adding a new column, that change is not dynamically reflected in the change table.
+For as long as the capture table continues to use the outdated schema, the {prodname} connector is unable to emit data change events for the table correctly.
+You must intervene to refresh the capture table to enable the connector to resume processing change events.   
+ 
+Because of the way that CDC is implemented in SQL Server, you cannot use {prodname} to update capture tables.
+To refresh capture tables, one must be a SQL Server database operator with elevated privileges can.
```

Comment:
Is this correct sentence?

Reply:
Thanks for catching that. I removed the extra word at the end of the sentence.